easybuild/easyconfigs/n/Nsight-Compute/Nsight-Compute-2022.1.0.eb

```diff
@@ -0,0 +1,30 @@
+##
+# Author:    Shahzeb Siddiqui <shahzebmsiddiqui@gmail.com>
+##
+easyblock = 'Binary'
+
+name = 'Nsight-Compute'
+version = '2022.1.0'
+
+homepage = 'https://developer.nvidia.com/nsight-compute'
+description = """NVIDIA® Nsight™ Compute is an interactive kernel profiler for CUDA applications. It provides detailed
+performance metrics and API debugging via a user interface and a command line tool. In addition, its baseline feature
+allows users to compare results within the tool. Nsight Compute provides a customizable and data-driven user interface
+and fast metric collection, which can be extended with rules-based analysis scripts for post-processing results. The
+rules-based guided analysis helps isolate and fix memory throughput, compute, and occupancy inefficiencies in the target
+application."""
+
+toolchain = SYSTEM
+
+# Download requires login https://developer.nvidia.com/downloads
```

Comment:
```suggestion
download_instructions = """Download %(name)s %(version)s from the NVIDIA website, which requires a NVIDIA developer account:
1. Go to https://developer.nvidia.com/gameworksdownload
2. Find %(name)s %(version)s
3. Download "%(name)s %(version)s (Linux)". This may require logging in to your NVIDIA developer account.
"""
```